### PR TITLE
feat: latest 12 into 12.1

### PR DESCRIPTION
**Pull request by mbritense**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -285,7 +285,6 @@
     "@valtimo/dashboard-management": "file:dist/valtimo/dashboard-management",
     "@valtimo/decision": "file:dist/valtimo/decision",
     "@valtimo/document": "file:dist/valtimo/document",
-    "@valtimo/zgw": "file:dist/valtimo/zgw",
```
Comment (teodor-ritense) [inline package.json:288]: Why is this removed?

**Review by sofiaIvarsRitense — APPROVED** [review]
